src/main/groovy/org/mule/tools/AssemblyContentsVerifierMojo.groovy

```diff
@@ -1,62 +1,63 @@
 package org.mule.tools
 
-import org.apache.maven.plugin.MojoFailureException
+import org.apache.maven.plugin.AbstractMojo
 import org.apache.maven.plugin.MojoExecutionException
+import org.apache.maven.plugin.MojoFailureException
+import org.apache.maven.plugins.annotations.Mojo
+import org.apache.maven.plugins.annotations.Parameter
 import org.apache.maven.project.MavenProject
-import org.codehaus.gmaven.mojo.GroovyMojo
+
 import java.util.regex.Pattern
 import java.util.zip.ZipFile
 
 /**
- * @goal verify
+ * Mojo for verifying contents of the output assembly against a controlled set of libraries.
```

Comment:
With "set of libraries" you mean the files inside the allowlist? in that case lest just say "set of files" or "against a controlled allowlist"

Reply:
Better now? 😅